src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
+  const newRobot = {};
+
+  for (let i = 0; i < keys.length; i++) {
+    newRobot[values[i]] = keys[i];
+  }
+
+  if (Object.keys(newRobot).length !== keys.length) {
+    return null;
+  }
```

Comment:
better to check inside loop if key exists instead of check after full iteration

Reply:
Please check